src/internet_identity/internet_identity.did

```diff
@@ -237,7 +237,12 @@ type CaptchaConfig = record {
 };
 
 // Init arguments of II which can be supplied on install and upgrade.
-// Setting a value to null keeps the previous value.
+//
+// Each field is wrapped is `opt` to indicate whether the field should
+// keep the previous value or update to a new value (e.g. `null` keeps the previous value).
+//
+// Some fields, like `openid_google`, have an additional nested `opt`,
+// this indicates enable/disable status (e.g. `opt null` disables a feature).
```

Comment:
Shall we add that `null` doesn't change the feature, whereas `opt null` disabled it.

Reply:
Oh, and thanks for adding this explanation!